src/main/webapp/projectContainer/ProjectContainer.js

```diff
@@ -113,6 +112,8 @@ export const ProjectContainer = hh(class ProjectContainer extends Component {
                 title: "Project Details",
               }, [
                   h(ProjectReview, {
+                    showSpinner: this.props.showSpinner,
+                    hideSpinner: this.props.hideSpinner,
```

Comment:
More of the same question here ... is it intentional that we pass the component's spinner down to these. I would imagine that it makes more sense to wrap ProjectReview with `LoadingWrapper`

Reply:
Now each component is wrapped by our LoadingWrapper. No need to pass anything through props.